scripts/bash/enceladus_env.template.sh

```diff
@@ -34,12 +34,14 @@ STD_DEFAULT_DRIVER_MEMORY=""
 STD_DEFAULT_DRIVER_CORES=""
 STD_DEFAULT_EXECUTOR_MEMORY=""
 STD_DEFAULT_EXECUTOR_CORES=""
+STD_DEFAULT_DRA_EXECUTOR_MEMORY=""
+STD_DEFAULT_DRA_EXECUTOR_CORES=""
```

Comment:
executor cores default should always be 1

Reply:
This can be done on the Enviroment Level

Reply:
I think 1 is a sensible default for the template.

Reply:
I am not arguing about the number. But I do think that this is not the place to put it. Either we come up with numbers for most of these in the template file or we leave them be, as we did so far and it is set on the servers.
I cannot see why to have a default for `dra_executor_cores` here but not for the 10 of the others.